cli/internal/taskhash/taskhash_test.go

```diff
@@ -82,24 +78,24 @@ func Test_manuallyHashPackage(t *testing.T) {
 		f.Close()
 	}
 	// now that we've created the repo, expect our .gitignore file too
-	files[turbopath.AnchoredUnixPath("libA/.gitignore")] = fileHash{contents: "", hash: "3237694bc3312ded18386964a855074af7b066af"}
+	files[turbopath.AnchoredUnixPath("child-dir/libA/.gitignore")] = fileHash{contents: "", hash: "3237694bc3312ded18386964a855074af7b066af"}
 
 	pkg := &fs.PackageJSON{
 		Dir: pkgName,
 	}
-	hashes, err := manuallyHashPackage(pkg, []string{}, turbopath.AbsoluteSystemPath(repoRoot.ToString()))
+	hashes, err := manuallyHashPackage(pkg, []string{}, repoRoot)
 	if err != nil {
 		t.Fatalf("failed to calculate manual hashes: %v", err)
 	}
-	prefix := pkgName + "/"
-	prefixLen := len(prefix)
```

Comment:
This test was accidentally platform dependent, but the upstream code appears to have gracefully handled the garbage in.